src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewCell.swift

```diff
@@ -95,16 +92,25 @@ extension DynamicCell {
 			   let cell = cell as? DynamicTableViewTextViewCell {
 				cell.configureAsLink(placeholder: placeHolder, urlString: text, font: font)
 			}
-			
-			if case .htmlString = cellStyle,
-			   let cell = cell as? DynamicTableViewTextViewCell {
-				cell.configure(htmlString: text)
-			}
-			
 			configure?(viewController, cell, indexPath)
 		}
 	}
-	
+
+	static func textWithLinks(
+		text: String,
+		links: [String: String],
+		linksColor: UIColor = .systemBlue,
```

Comment:
```suggestion
		linksColor: UIColor = .enaColor(for: .textTint),
```

Think it would make sense to use our `.textTint` color by default